utils/auth_jwt.ts

```diff
@@ -62,3 +69,36 @@ export const getValidateAuthJWT = ({
         TE.filterOrElse(AuthJWT.is, () => E.toError("Invalid AuthJWT payload"))
       )
   );
+
+/**
+ * A middleware that verify the jwt
+ *
+ * @param jwtConfig the config for the jwt
+ * */
+
+export const verifyJWTMiddleware = (
+  jwtConfig: JWTConfig
+): IRequestMiddleware<"IResponseErrorValidation", AuthJWT> => (
+  req
+): Promise<E.Either<IResponseErrorValidation, AuthJWT>> =>
+  pipe(
+    req.headers["x-pagopa-lollipop-auth"],
```

Comment:
The header name should came from the configuration.
This change will make the `verifyJWTMiddleware` usable to validate any JWT